client/components/plans/plan-pill/index.jsx

```diff
@@ -1,7 +1,11 @@
 import './style.scss';
 
 export default ( props ) => (
-	<div className={ `plan-pill${ props.isInSignup ? ' is-in-signup' : '' }` }>
+	<div
```

Comment:
This might look more readable if we're using the classNames package:
(Code below is not actually tested yet)

```
import classNames from 'classnames';

...

className={classNames('plan-pill', {
      'is-in-signup': props.isInSignup,
      'is-current-plan': props.isCurrentPlan
  })
}
```